bin/gollum

```diff
@@ -135,6 +135,9 @@ MSG
     "By default, uses the 'TeX-AMS-MML_HTMLorMML' config with the 'autoload-all' extension.") do
     wiki_options[:mathjax] = true
   end
+  opts.on("--critic-markup", "Enable CriticMarkup support.") do
```

Comment:
Perhaps something more descriptive for people who don't know CM yet:
`Enable support for annotations using CriticMarkup`
?